.github/workflows/publish.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - 'DOI-759-ev-dashboard-client-npm-publish'
+  pull_request:
+    branches:
+      - '*'
+jobs:
+  cancel-previous:
```

Comment:
Just curious, what does this job do? It lets you cancel a package publish while it is happening?

Reply:
This action cancels previous runs for one or more branches/prs associated with a workflow, effectively limiting the resource consumption of the workflow to one per branch.
https://github.com/marketplace/actions/cancel-previous-workflow-runs